### PR TITLE
[dev][setup] Uncomment the protoc dependency installation in dev setup.

### PR DESCRIPTION
**Pull request by larry-aptos**
### Description
* Previously, `protoc` installation is removed due to bad release upstream. Uncomment it now since the error is fixed upstream.

### Test Plan

Tested on a fresh ubuntu machine. Looks good overall.

image

### CODE REVIEW

**Review by rustielin — APPROVED** [review]
LGTM

Is there a way to pin these deps btw?

File: scripts/dev_setup.sh [hunk 1]
```diff
@@ -147,17 +147,16 @@ function install_protoc {
...
-  # fi
-  # if ! command -v protoc-gen-prost-serde &> /dev/null; then
-  #   cargo install protoc-gen-prost-serde
-  # fi
-  # if ! command -v protoc-gen-prost-crate &> /dev/null; then
-  #   cargo install protoc-gen-prost-crate
-  # fi
+  # Install the cargo plugins
+  if ! command -v protoc-gen-prost &> /dev/null; then
+    cargo install protoc-gen-prost --locked
```
Comment (ibalajiarun) [inline scripts/dev_setup.sh:152]: can we add `--version` flag to pin the version?

Comment (larry-aptos) [reply scripts/dev_setup.sh:152]: The issue is from indirect dependency; pinning to a specific version doesn't help us avoid the issue tho.

**Review by bowenyang007 — APPROVED** [review]

**Comment by larry-aptos** [comment]
It's now pinning to a good version that repo used now.  This is future-proof in `protoc-gen-prost` case.

**Pull request merged**